org.mixedrealitytoolkit.input/InteractionModes/IInteractionModeDetector.cs

```diff
@@ -31,6 +32,13 @@ public interface IInteractionModeDetector
         /// Get a list of the <see cref="GameObject"/> instances which represent the controllers that this interaction mode detector has jurisdiction over.
         /// </summary>
         /// <returns>The list of the <see cref="GameObject"/> instances which represent the controllers that this interaction mode detector has jurisdiction over.</returns>
+        [Obsolete("This function is obsolete and will be removed in a future version. Please use GetInteractorGroups instead.")]
```

Comment:
We should probably create a new interface and obsolete this one entirely unless we're planning to remove this `Obsolete` method before shipping.

Otherwise, any developer who wants to inherit from this interface is forced to "implement" an obsolete method, and any developer who already has won't be able to update their class cleanly.